src/contour/Config.h

```diff
@@ -936,13 +935,15 @@ struct Writer
     template <typename T>
     auto format(T v)
     {
-        return fmt::format("{}", v);
+        return std::format("{}", v);
     }
 
     template <typename... T>
     [[nodiscard]] std::string format(std::string_view doc, T... args)
     {
-        return fmt::format(fmt::runtime(doc), format(args)..., fmt::arg("comment", "#"));
+        std::string replacedDoc =
```

Comment:
i think we need to use `replaceCommentPlaceholder` function here 
Also, why `vformat`?

Reply:
I tried also without `vformat`, but then the formatting string needs to be a literal (i.e. `std::format_string<Args...>`)

Reply:
Oh, I was hoping that they made it into c++23, but only in c++26 https://en.cppreference.com/w/cpp/utility/format/runtime_format